packages/textlint-formatter/test/formatters/checkstyle.js

```diff
@@ -4,13 +4,13 @@
  */
 
 "use strict";
+import formatter from "../../src/formatters/checkstyle";
 
 //------------------------------------------------------------------------------
 // Requirements
 //------------------------------------------------------------------------------
 
-var assert = require("power-assert"),
-    formatter = require("../../lib/formatters/checkstyle");
+var assert = require("power-assert");
```

Comment:
Replace to Node's `assert`.

```
import * as assert from "assert"
```

Because, power-assert does not work in current config.
It need to additional transpiler like `intelli-espower-loader`.
We can simply drop power-assert instead of adding more transpile process.